apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -322,6 +328,15 @@ const styles = {
     flexDirection: 'row',
     marginTop: 40
   },
+  icon: {
+    marginRight: 4,
+    marginLeft: 2,
```

Comment:
Double check that this spacing looks good for right-to-left languages as well.

Reply:
Thanks for this- I updated it and added a photo in the PR description